<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>258</width>
     <height>485</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="totalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="numOfPeoLabel">
        <property name="text">
         <string>人数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="numOfPeoSpin"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="moEachPeoSpin">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="moEachPeoLabel">
        <property name="text">
         <string>每个人多少钱</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="foodsTable"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>菜单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="confiremButton">
      <property name="text">
       <string>生成菜单</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
